src/navsat_transform.cpp

```diff
@@ -93,9 +107,11 @@ void NavSatTransform::run()
   node_->get_parameter_or("frequency", frequency, 10.0);
   node_->get_parameter_or("delay", delay, 0.0);
   node_->get_parameter_or("transform_timeout", transform_timeout, 0.0);
+  queue_size_ = node_->declare_parameter("queue_size", 10);
```

Comment:
You should make clear what this queue size is for, since there's alot of parameterizable queue sizes in this package. 

Edit: looking down the review I think this is mostly a miscommunication on how the `XYZ_queue_size` parameter being used. In any case, all the member variables `queue_size_` introduced should be removed

Reply:
ok will remove